doc/tutorials/enable_s5.rst

```diff
@@ -31,135 +31,199 @@ The diagram below shows the overall architecture:
 
-  .. code-block:: bash
-
-     # Power Management (PM) configuration using vUART channel
-     pm_channel="--pm_notify_channel uart"
-     pm_by_vuart="--pm_by_vuart pty,/run/acrn/life_mngr_"$vm_name
-     pm_vuart_node="-s 1:0,lpc -l com2,/run/acrn/life_mngr_"$vm_name
+   .. code-block:: none
 
-     acrn-dm -A -m $mem_size -s 0:0,hostbridge \
-       ...
-       $pm_channel \
-       $pm_by_vuart \
-       $pm_vuart_node \
-       ...
+      cd acrn-hypervisor
+      make life_mngr
 
-* For RT-Linux, include these lines in the launch script:
+#. For Service VM, LaaG VM and RT-Linux VM, run the Lifecycle Manager daemon:
 
-  .. code-block:: bash
+   a. Copy ``life_mngr.conf``, ``s5_trigger_linux.py``, ``user_vm_shutdown.py``, ``life_mngr`` and ``life_mngr.service``
+      into the Service VM and User VMs.
 
-     # Power Management (PM) configuration
-     pm_channel="--pm_notify_channel uart"
-     pm_by_vuart="--pm_by_vuart tty,/dev/ttyS1"
+      .. code-block:: none
 
-     /usr/bin/acrn-dm -A -m $mem_size -s 0:0,hostbridge \
-        ...
-        $pm_channel \
-        $pm_by_vuart \
-        ...
+         scp build/misc/services/s5_trigger_linux.py root@<target board address>:~/
+         scp build/misc/services/life_mngr root@<target board address>:/usr/bin/
+         scp build/misc/services/life_mngr.conf root@<target board address>:/etc/life_mngr/
+         scp build/misc/services/life_mngr.service root@<target board address>:/lib/systemd/system/
 
-  .. note:: For RT-Linux, the vUART is emulated in the hypervisor; expose the node as ``/dev/ttySn``.
+         scp misc/services/life_mngr/user_vm_shutdown.py root@<target board address>:~/
 
-#. For LaaG and RT-Linux VMs, run the lifecycle manager daemon:
+      .. note:: user_vm_shutdown.py is only needed to be copied into Service VM.
 
-   a. Use these commands to build the lifecycle manager daemon, ``life_mngr``.
+   #. Edit options in ``/etc/life_mngr/life_mngr.conf`` in the Service VM.
 
       .. code-block:: none
 
-         $ cd acrn-hypervisor
-         $ make life_mngr
+         VM_TYPE=service_vm
+         VM_NAME=Service_VM
+         DEV_NAME=tty:/dev/ttyS8,/dev/ttyS9,/dev/ttyS10,/dev/ttyS11,/dev/ttyS12,/dev/ttyS13,/dev/ttyS14
+         ALLOW_TRIGGER_S5=/dev/ttySn
 
-   #. Copy ``life_mngr`` and ``life_mngr.service`` into the User VM:
+      .. note:: The mapping between User VM ID and communication serail device name (/dev/ttySn)
+                  in the /etc/serial.conf. If /dev/ttySn is configured in the  ``ALLOW_TRIGGER_S5``,
+                  this means system shutdown is allowed to be triggered in the the corresponding User VM.
+
+   #. Edit options in ``/etc/life_mngr/life_mngr.conf`` in the User VM.
 
       .. code-block:: none
 
-         $ scp build/misc/services/life_mngr root@<test board address>:/usr/bin/life_mngr
-         $ scp build/misc/services/life_mngr.service root@<test board address>:/lib/systemd/system/life_mngr.service
+         VM_TYPE=user_vm
```

Comment:
Do we describe anywhere what the different values for this field can be?

Reply:
Thanks! More details are described in the "/etc/life_mngr/life_mngr.conf".